portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -23,10 +24,24 @@
 /** Servlet that returns some example content. TODO: modify this file to handle comments data */
 @WebServlet("/data")
 public class DataServlet extends HttpServlet {
-
+  private final ArrayList<String> messages = new ArrayList<String>();
```

Comment:
nit: you can replace `new ArrayList<String>()` with `new ArrayList<>`, as the type param is automatically inferred from the declaration.

Reply:
Can you still do this in this PR?